src/Raygun4Aspire/RaygunAspireWebAppBuilderExtensions.cs

```diff
@@ -10,9 +12,9 @@ public static IResourceBuilder<RaygunAspireWebAppResource> AddRaygun(this IDistr
     {
       var raygun = new RaygunAspireWebAppResource(name);
       return builder.AddResource(raygun)
-                    .WithAnnotation(new EndpointAnnotation(ProtocolType.Tcp, uriScheme: "http", port: port ?? DefaultHostPort, containerPort: 8080))
-                    .WithAnnotation(new ContainerImageAnnotation { Image = "raygunowner/raygun-aspire-portal", Tag = "1.0.0-preview.1.0.0" })
-                    .WithVolumeMount("raygun-data", "/app/raygun")
+                    .WithAnnotation(new ContainerImageAnnotation { Image = "raygunowner/raygun-aspire-portal", Tag = "1.0.0" })
+                    .WithAnnotation(new EndpointAnnotation(ProtocolType.Tcp, uriScheme: "http", port: port ?? DefaultHostPort, targetPort: 8080))
```

Comment:
Curious, we defined the port as `24605`, is there no way for Aspire to allocate a port to avoid conflicts between 2 projects picking the same port?

Reply:
Couldn't see a way for the Raygun4Aspire to discover a random port assigned by Aspire - could revisit in a future version.